config/main

```diff
@@ -0,0 +1 @@
+include config/default
```

Comment:
Please note that in UI or API, no config file directly extend ``config/default`` but ``Makefile``:
https://github.com/c2corg/v6_api/blob/master/config/main#L1

``Makefile`` itself includes the default config:
https://github.com/c2corg/v6_api/blob/master/Makefile#L9

Shouldn't we do the same here for consistency reasons?